File: CS/WindowsFormsApplication88/CustomInnerColorPickControl.cs
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.ViewInfo;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace WindowsFormsApplication88 {
    public class CustomInnerColorPickControl : InnerColorPickControl {
        public CustomInnerColorPickControl() : base() {

        }
        protected override BaseStyleControlViewInfo CreateViewInfo() {
            return new CustomInnerColorPickControlViewInfo(this);
        }
    }
}

File: VB/WindowsFormsApplication88/CustomInnerColorPickControl.vb
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.ViewInfo
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace WindowsFormsApplication88
    Public Class CustomInnerColorPickControl
        Inherits InnerColorPickControl

        Public Sub New()
            MyBase.New()

        End Sub
        Protected Overrides Function CreateViewInfo() As BaseStyleControlViewInfo
            Return New CustomInnerColorPickControlViewInfo(Me)
        End Function
    End Class
End Namespace
